docs/UpdatedData.md

```diff
@@ -241,7 +247,7 @@ In the below example, The 2 objects are sent in 2 separate requests. The first o
 Leveraging the `deviceEvent` type with the already defined `alarm` subType. We add couple of fields to get more details on alarms and acknowledgement. 
 
 The below fields are mandatory if `alarmType` is set to `handset`. They remain optional for any other values.
-- `eventId`: unique Id of the event generated by the client system. This ID will be used to reconciliate data for the same event. _Maximum length is 64 characters_.
+- `guid` (replacing `eventId` deprecated field): unique Id of the event generated by the client system. This ID will be used to reconciliate data for the same event. _Maximum length is 64 characters_.
```

Comment:
this ID will be used with combination of DeviceId , userId to reconciliate data for the same event

Reply:
Done